common/workunit/workunit.cpp

```diff
@@ -1830,6 +1847,7 @@ mapEnums workunitSortFields[] =
    { WUSFwuidhigh, "@" },
    { WUSFwildwuid, "@" },
    { WUSFappvalue, "Application" },
+   { WUSFfilewritten, "Files/File/@name" },
```

Comment:
The 'sort' fields are used for both the sort order and the filters, with only a subset actually permissible as sort orders. This is not a great interface, but it's not a new thing...